src/box/errcode.h

```diff
@@ -115,286 +115,286 @@ struct errcode_record {
 	_(ER_WAL_IO, 40,			"Failed to write to disk") \
-	_(ER_MORE_THAN_ONE_TUPLE, 41,		"Get() doesn't support partial keys and non-unique indexes") \
-	_(ER_ACCESS_DENIED, 42,			"%s access to %s '%s' is denied for user '%s'") \
-	_(ER_CREATE_USER, 43,			"Failed to create user '%s': %s") \
-	_(ER_DROP_USER, 44,			"Failed to drop user or role '%s': %s") \
-	_(ER_NO_SUCH_USER, 45,			"User '%s' is not found") \
-	_(ER_USER_EXISTS, 46,			"User '%s' already exists") \
+	_(ER_MORE_THAN_ONE_TUPLE, 41,		"Get() does not support partial keys and non-unique indexes") \
+	_(ER_ACCESS_DENIED, 42,			"Access denied") \
+	_(ER_CREATE_USER, 43,			"Failed to create user", "user", STRING, "details", STRING) \
+	_(ER_DROP_USER, 44,			"Failed to drop user", "user", STRING, "details", STRING) \
+	_(ER_NO_SUCH_USER, 45,			"User does not exist", "user", STRING) \
+	_(ER_USER_EXISTS, 46,                   "User already exists", "user", STRING) \
 	_(ER_CREDS_MISMATCH, 47,		"User not found or supplied credentials are invalid") \
-	_(ER_UNKNOWN_REQUEST_TYPE, 48,		"Unknown request type %u") \
-	_(ER_UNKNOWN_SCHEMA_OBJECT, 49,		"Unknown object type '%s'") \
-	_(ER_CREATE_FUNCTION, 50,		"Failed to create function '%s': %s") \
-	_(ER_NO_SUCH_FUNCTION, 51,		"Function '%s' does not exist") \
-	_(ER_FUNCTION_EXISTS, 52,		"Function '%s' already exists") \
+	_(ER_UNKNOWN_REQUEST_TYPE, 48,		"Unknown request type", "request_type", UINT) \
+	_(ER_UNKNOWN_SCHEMA_OBJECT, 49,		"Unknown object type", "object_type", STRING) \
+	_(ER_CREATE_FUNCTION, 50,		"Failed to create function", "function", STRING, "details", STRING) \
+	_(ER_NO_SUCH_FUNCTION, 51,		"Function does not exist", "function", STRING) \
+	_(ER_FUNCTION_EXISTS, 52,		"Function already exists", "function", STRING) \
 	_(ER_BEFORE_REPLACE_RET, 53,		"Invalid return value of space:before_replace trigger: expected tuple or nil") \
-	_(ER_MULTISTATEMENT_TRANSACTION, 54,	"Can not perform %s in a multi-statement transaction") \
-	_(ER_TRIGGER_EXISTS, 55,		"Trigger '%s' already exists") \
-	_(ER_USER_MAX, 56,			"A limit on the total number of users has been reached: %u") \
-	_(ER_NO_SUCH_ENGINE, 57,		"Space engine '%s' does not exist") \
-	_(ER_RELOAD_CFG, 58,			"Can't set option '%s' dynamically") \
-	_(ER_CFG, 59,				"Incorrect value for option '%s': %s") \
+	_(ER_MULTISTATEMENT_TRANSACTION, 54,	"Can not perform in a multi-statement transaction", "action", STRING) \
+	_(ER_TRIGGER_EXISTS, 55,		"Trigger already exists", "trigger", STRING) \
+	_(ER_USER_MAX, 56,			"A limit on the total number of users has been reached", "limit", UINT) \
+	_(ER_NO_SUCH_ENGINE, 57,		"Space engine does not exist", "engine", STRING) \
+	_(ER_RELOAD_CFG, 58,			"Can't set option dynamically", "option", STRING) \
+	_(ER_CFG, 59,				"Incorrect value for option", "option", STRING, "details", STRING) \
 	_(ER_SAVEPOINT_EMPTY_TX, 60,		"Can not set a savepoint in an empty transaction") \
```

Comment:
"Can not" -> "Cannot"

Reply:
Fixed as part of work for previous comment.